pkg/utils/oci/helmregistry.go

```diff
@@ -25,7 +25,7 @@ import (
 const (
 	mediaTypeHelm = "application/vnd.cncf.helm.chart.content.v1.tar+gzip"
 
-	inKubernetesRegistry = "garden.local.gardener.cloud:5001"
+	inKubernetesRegistry = "local.gardener.cloud:"
```

Comment:
Could you elaborate what is the motivation and the impact of this change?

Reply:
OK, got it. Based on this constant, there is only a check whether to use TLS: https://github.com/gardener/gardener/blob/1325dea3f4aa0f1ef3bc24c0fae56cc1035d4150/pkg/utils/oci/helmregistry.go#L124-L127

I thought that we change from `garden.local.gardener.cloud:5001` to `registry.local.gardener.cloud:5001` in this PR. 

Thanks for explaining!